### DISCUSSION TITLE
hey, super noob question but I've been away from servers and ssh for a very long time and need some help

### DISCUSSION THREAD

**Discussion by osnapitsjoey** [gtxaspec/wz_mini_hacks#526] [Q&A]
so I created ssh files on my unraid server, opened the .pub file and pasted it into the wz_mini spot, but am i suppose to be using my unraids server ip address to access? It just seems as though im ssh-ing into my own server from my own server

**Comment by bjs-pdx** [comment]
You should be creating your keys for you, as a user, not for a machine.  Not sure which OS you use but there are a whole set of instructions telling you how to create the keys based on your main computer OS.

**Comment by osnapitsjoey** [comment]
I've created them for me, but on unraid. I think something is up though as
port 22 is refusing

**Reply by bjs-pdx** [reply]
Two possibilities.  One, SSH isn't running on the camera.  Two, there is something wrong with your key.  What OS are you running on your computer/laptop?  What OS did you use to copy the key to the sd card?  Windows changes the file to have carriage return and line feeds, linux I think is just carriage returns. 


Take a look here as well:  https://averagelinuxuser.com/how-to-use-public-key-authentication/

**Comment by osnapitsjoey** [comment]
Oh ssh isn't enabled by default? I didn't see it in the config file. I
created on unraid the webgui on my chromebook, and then copy pasted the pub
key and placed pasted it into the auth place using my chromebook

**Reply by bjs-pdx** [reply]
SSH is enabled by default, but you need your key on the sd card.  This section of the readme covers ssh:

https://github.com/gtxaspec/wz_mini_hacks#setup-v3panv2
```

Setup v3/PANv2

    git clone the repo or download the repo zip
    perform a fresh format on your micro-sd card, using fat-32 ( this is a hard requirement, the bootloader does not support ex-fat or ext, and thus will not load wz_mini ), DOS partition map type, volume name does not matter.
    copy all the files inside of SD_ROOT to your micro sd card
    SSH is enabled, but is secured using public key authentication for security. Edit the file wz_mini/etc/ssh/authorized_keys and enter your public key here. If you need a simple guide, [how to use public key authentication](https://averagelinuxuser.com/how-to-use-public-key-authentication/)
```

**Reply by osnapitsjoey** [reply]
hey thanks! i accidently thru the entire sd_root folder in there instead of extracting to lol. It works now!

so now that I have ssh connection, what firmware should I load onto it?

**Reply by bjs-pdx** [reply]
Which camera?

**Comment by osnapitsjoey** [comment]
I will double check the fat32 formatting, but I did follow that. I was a
bit tired from a long day of work though. I really appreciate you helping!

**Comment by osnapitsjoey** [comment]
Wyze v3

**Reply by bjs-pdx** [reply]
Try this post:  https://github.com/gtxaspec/wz_mini_hacks/issues/187

This seems to be the most recommended version:  4.36.9.139 is a pretty solid firmware version anyway
